include/zephyr/net/wifi_nm.h

```diff
@@ -94,6 +94,10 @@ int wifi_nm_unregister_mgd_iface(struct wifi_nm_instance *nm, struct net_if *ifa
  * @}
  */
 
+#define SSID_MAX_LEN 32
```

Comment:
There is already `WIFI_SSID_MAX_LEN` in wifi.h, please use that

Reply:
Done